collection-scripts/gather_ceph_resources

```diff
@@ -388,42 +388,50 @@ for ns in $namespaces; do
                     dbglog "waiting for ${pids_rbd[*]} to terminate"
                     wait "${pids_rbd[@]}"
                 fi
-                find "${COMMAND_OUTPUT_DIR}" -name "rbd_mirror_image_status_*.part" -print0 | xargs -0 cat >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status
+                find "${COMMAND_OUTPUT_DIR}" -name "rbd_mirror_image_status_*.part" -print0 | xargs -0 cat >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${bp}"
                 find "${COMMAND_OUTPUT_DIR}" -name "rbd_mirror_image_status_*.part" -print0 | xargs -0 rm -f
                 # Collecting mirror image status for provider mode with rados namespace
                 rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls") 
                 for rns in $rados_namespaces; do
```

Comment:
This will include the title NAME.

```
$ rbd -p ocs-storagecluster-cephblockpool namespace ls
NAME                                               
cephradosnamespace-e71b5a2df7098e4e8682f7a0f881f5a2
```

Reply:
it will be eliminated in next line while checking for mirroring enabled/disabled, not sure if there is any other way